### DISCUSSION TITLE
1.26.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#13005] [General]
# Features
- #12731 Allow to create backup via `zigbee2mqtt/bridge/request/backup` and frontend (*Settings -> Tools -> Request data folder backup*)
- #12868 Add group `off_state` option

# Improvements
- [#4309](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4309) Expose battery voltage for Perenio PECWS01
- #11469 Expose `energy`, `current` and `voltage` for Sercomm SZ-ESW01-AU
- [#3401](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4301) Expose `power_on_behavior` for TuYa TS0004
- [#4325](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4325) Expose `occupied_cooling_setpoint` for Zen Zen-01-W
- [#4324](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4324) Expose `report_delay` for EFEKTA_miniPWS
- [#4329](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4329) Send hourly data updates of Xiaomi GZCGQ01LM
- [#4310](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4310) Expose `battery` and `battery_low` for TuYa S-LUX-ZB (`_TZE200_pisltm67`)
- #12693 Send cached group state on startup and when Home Assistant comes online
- [#4344](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4344) Expose `battery` for Develco WISZB-120
- #12687 Expose `energy` for Xiaomi QBKG11LM and QBKG12LM
- #11738 Change Home Assistant `hold_mode` to `preset_mode` and expose `away_mode` as separate switch
- [#4352](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4352) Add and correct `whiteLabel` for some Xiaomi devices
- [#4353](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4353) Support ota for TS011F_din_smart_relay
- #10303 Expose `lift_duration` for Schneider Electric S520567
- [#551](https://github.com/Koenkk/zigbee-herdsman/pull/551) Support touchlink for EZSP adapter
- [#4369](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4369) Expose `side` for Xiaomi MFKZQ01LM
- #12904 Allow to specify MQTT server in secrets
- #12919 Move `_TZ3210_rcggc0ys` to Moes ZLD-RCW

# Fixes
- #12666 Fix power source of Third Reality 3RDS17BZ and 3RMS16BZ incorrect
- #12677 Fix `_TZE200_sbordckq` not recognized anymore
- [#4308](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4308) Fix `adaptation_run_control ` values for Danfoss 014G2461
- [#4310](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4310) Fix `illuminance_lux` and `battery` values being a string instead of number for TuYa S-LUX-ZB
- #12666 Fix battery percentage of Third Reality 3RMS16BZ incorrect
- #7405 Identify `_TZ3000_v4mevirn` as ZIGBEE-B09-UK
- [#4284](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4284) Fix `running_state` for Centralite 3156105
- #12701 Fix whitelabel `CCT Light` not being recognised anymore
- #12651 Fix power source of Müller Licht 404022
- #12706 Add back `pin_code` expose for Danalock V3-BTZB/V3-BTZBE
- #12734 Fix not able to individually control outputs of LELLKI WP33-EU/WP34-EU
- #12736 Fix not able to individually control outputs of Zemismart ZIGBEE-B09-UK
- #12740 Disable OTA for Sengled E1C-NB7 since none is available
- #12749 Fix `color_temperature` not exposed for Philips 929002376801
- #12719 Fix TuYa `TS0601_cover` not reporting position
- #4310 Fix TuYa S-LUX-ZB `battery_low` being a string instead of boolean
- #12750 Fix Yale YDF40 interview failing
- #12811 Fix Moes BHT-002-GCLZB (`_TZE200_ztvwu4nk`) temperature divided by 10
- #12163 Fix battery not exposed for Woox R7060
- #12846 Fix Philips 929002376101 not exposing color temperature 
- #9847 Fix temperature: null published for ORVIBO ST30
- [#4356](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4356) Expose `led_disabled_night`, `mode_switch`, `power_outage_count` and `device_temperature` for Xiaomi WS-EUK01 and WS-EUK02
- [#4365](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4365) Fix Envilar ZG50CC-CCT-DRIVER not exposing color temperature
- #12899 Fix OTA not enabled for some Philips Hue lights
- [#4376](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4376) Move `_TZ3000_nzkqcvvs` from `TS011F_plug_1` to `TS011F_plug_2` since it doesn't support energy measurements
- [#4379](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4379) Expose `power_outage_count` for more Xiaomi devices
- #12911 Fix setting some settings via environment variable not working (e.g. `ZIGBEE2MQTT_CONFIG_FRONTEND_URL`)
- #12957 Fix RGB Genie missing actions for master on/off buttons
- [#4387](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4387) Fix no battery value for TuYa WSD500A
- [#4385](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4385) Fix TuYa `power_outage_memory` converter
- #12588  Fix no `energy` and `power` values for Jasco 43132

# New supported devices
This release adds support for **32** new devices:
- `5047231P6` Philips Hue White ambiance Buckram double spotlight
- `5047331P6` Philips Hue White ambiance Buckram triple spotlight
- `552-72201` Niko Connectable dimmer
- `552-72301` Niko Connectable motor control
- `9290018609` Philips Hue White E26 806 lumen
- `929002376703` Philips Hue Iris silver special edition (generation 4) 
- `94191` Paulmann Plug & shine LED strip
- `AC23684` LEDVANCE Classic E26 tunable white 9w 800 lumen
- `BSEED_TS0601_cover` BSEED Zigbee curtain switch
- `D086-ZG` HZC Electric Zigbee dual dimmer
- `HG08010` Lidl Livarno Home outdoor spotlight
- `HG08131C` Lidl Livarno Home outdoor E27 bulb in set with flare
- `HG08383B` Lidl Livarno outdoor LED light chain
- `HS1CG` Heiman Smart combustible gas sensor
- `LDSENK09` ADEO Security system key fob
- `MP-840` Visonic Long range pet immune PIR motion sensor
- `PC321` OWON 3-Phase clamp power meter
- `QT06` GiEX Water irrigation valve
- `RMC002` Bouffalolab US plug smart socket
- `ROB_200-030-0` ROBB Zigbee AC in wall switch 400W (2-wire)
- `S40ZBTPB` SONOFF 15A Zigbee smart plug
- `SMART-SMOKE10` Alecto Smoke detector
- `SR-ZGP2801K-5C` Sunricher Pushbutton transmitter module
- `SRB01` Evvr In-wall relay switch
- `TB26-4` Zemismart 4-gang smart wall switch
- `TB26-6` Zemismart 6-gang smart wall switch
- `UFO-R11` Moes Universal smart IR remote control
- `ZB-Sm` TuYa Tubular motor
- `ZNCLBL01LM` Xiaomi Aqara curtain driver E1
- `ZS06` TuYa Universal smart IR remote control
- `ZSDR-850` Trust Smoke detector
- `ZXZIR-02` AUBESS Universal smart IR remote control

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.26.0'>1.26.0</a>.</em>

**Comment by MichalKrasowski** [comment]
Broken for me.

s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/socat.sh
[22:28:27] INFO: Socat not enabled, marking service as down
cont-init: info: /etc/cont-init.d/socat.sh exited 0
cont-init: info: running /etc/cont-init.d/zigbee2mqtt.sh
cont-init: info: /etc/cont-init.d/zigbee2mqtt.sh exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun socat (no readiness notification)
services-up: info: copying legacy longrun zigbee2mqtt (no readiness notification)
[22:28:34] INFO: Handing over control to Zigbee2MQTT Core ...
> zigbee2mqtt@1.26.0 start
> node index.js
/app/dist/util/externally-loaded.js:62
            .withAwayMode()
             ^
TypeError: exposes.climate(...).withSetpoint(...).withLocalTemperature(...).withSystemMode(...).withLocalTemperatureCalibration(...).withAwayMode is not a function
    at /app/dist/util/externally-loaded.js:62:14
    at Script.runInContext (node:vm:139:12)
    at Script.runInNewContext (node:vm:144:17)
    at Object.runInNewContext (node:vm:298:38)
    at loadModuleFromText (/app/lib/util/utils.ts:148:8)
    at loadModuleFromFile (/app/lib/util/utils.ts:155:12)
    at Object.getExternalConvertersDefinitions (/app/lib/util/utils.ts:165:25)
    at getExternalConvertersDefinitions.next (<anonymous>)
    at new ExternalConverters (/app/lib/extension/externalConverters.ts:12:20)
    at new Controller (/app/lib/controller.ts:84:58)

**Reply by MichalKrasowski** [reply]
Z2M being unable to boot means death to my plants :(

**Reply by Koenkk** [reply]
You are using an external converter where `withAwayMode()` is used, this has been removed so you should update your external converter.

**Reply by MichalKrasowski** [reply]
Nope :(

data_path: /share/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/dev/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false
  restartdelay: 1
  initialdelay: 1
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://core-mosquitto
  user: ######
  password: #####
serial:
  port: /dev/ttyUSB0
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
advanced:
  log_level: info
  pan_id: 6754
  channel: 11
  network_key:
####
  availability_blocklist: []
  availability_passlist: []
device_options: {}
blocklist: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental:
  new_api: true

**Reply by MichalKrasowski** [reply]
Partially restored backup to downgrade Z2M to 1.25.2, good as new.

1.26.0 unfortunately still broken for me 💔

**Reply by Koenkk** [reply]
check if there are external converters in `/share/zigbee2mqtt/configuration.yaml`.

**Reply by tonyt2975** [reply]
This was the problem I was having, this solution worked for me.  Thank you

**Reply by MichalKrasowski** [reply]
Ok, I've verified the existance of /share/zigbee2mqtt/configuration.yaml, indeed the external converter was there, removed it, works now.

**Comment by chersoph** [comment]
After uppdate M2Z will not start.
info  2022-07-27 11:22:55: Logging to console and directory: '/config/zigbee2mqtt/log/2022-07-27.11-22-54' filename: log.txt
info  2022-07-27 11:22:55: Starting Zigbee2MQTT version 1.26.0 (commit #unknown)
info  2022-07-27 11:22:55: Starting zigbee-herdsman (0.14.40)
error 2022-07-27 11:23:07: Error while starting zigbee-herdsman
error 2022-07-27 11:23:07: Failed to start zigbee
error 2022-07-27 11:23:07: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 2022-07-27 11:23:07: Exiting...
error 2022-07-27 11:23:07: undefined

Addon configuration (unchanges since upgrade)
data_path: /config/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: '-d -d'
  log: false
mqtt: {}
serial:
  port: /dev/ttyS0
  adapter: deconz

configuration.yaml
homeassistant: true
mqtt:
  server: mqtt://core-mosquitto:1883
  user: ####
  password: #####
serial:
  port: /dev/ttyS0
  adapter: deconz
frontend:
  port: 8099
devices:

groups:

permit_join: false
